src/ui/public/kibiter/menu/custom_menu.js

```diff
@@ -0,0 +1,56 @@
+
+export function customMenu($scope) {
+    // Define the number of columns
+    let numberColumns = 4
+
+    $scope.showInfo = function (item) {
```

Comment:
This is the old style javascript for defining functions. Change it to the new style.